sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -26,6 +27,7 @@ import (
 	"github.com/apache/beam/sdks/go/pkg/beam/core/util/protox"
 	"github.com/apache/beam/sdks/go/pkg/beam/internal/errors"
 	pb "github.com/apache/beam/sdks/go/pkg/beam/model/pipeline_v1"
+	"github.com/apache/beam/sdks/go/pkg/beam/options/jobopts"
```

Comment:
I don't like that we're importing this package here. jobopts creates flags, which could be unexpected or unnecessary for certain runners.

Could we instead just pass the environment urn in as a string? (the only thing we actually use from the package).

Reply:
OK, done.